### DISCUSSION TITLE
Welcome to password-strength-meter Discussions!

### DISCUSSION THREAD

**Discussion by antoantonyk** [antoantonyk/password-strength-meter#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kevinch3** [comment]
Hello! I'm using the library to ensure the input of a project. I'm wondering if will support IVY? Amazing snippet!

**Reply by somnath-git** [reply]
raised pr

**Comment by antoantonyk** [comment]
@kevinch3  yes. Please use the latest version
